localstripe/resources.py

```diff
@@ -678,6 +678,11 @@ def _api_delete(cls, id):
         schedule_webhook(Event('customer.deleted', obj))
         return super()._api_delete(id)
 
+    @classmethod
+    def _api_retrieve_sources(cls, id):        
```

Comment:
To be consistent with the rest of the code, maybe `_api_list_sources` would be better.

Reply:
Changed